config/invenio.conf

```diff
@@ -492,6 +492,12 @@ CFG_WEBSEARCH_RSS_I18N_COLLECTIONS =
 # a feed cache is valid.
 CFG_WEBSEARCH_RSS_TTL = 360
 
+# CFG_WEBSEARCH_INSTITUTION_COLLECTIONS -- comma-separated list of
+# collections that contains records of institutions.
+# It is used by BibIndexCountryTokenizer to know in which collections the
+# institution's name should be searched.
+CFG_WEBSEARCH_INSTITUTION_COLLECTIONS = Institutes,Authority Institution,Institutions
```

Comment:
We only have "Institutes" as the demo collection name.  The others could be set in INSPIRE overlay.

BTW, based on the CFG description, shouldn't it be named `s/CFG_WEBSEARCH_/CFG_BIBINDEX/`?  But then again we've been moving all configurations into DB, deprecating ones in `invenio.conf`, which this addition seems to go against...